### PR TITLE
Supplemental work regarding chapter and section notes

### PR DESCRIPTION
**Pull request by jarl-borg**
Per @matthewford note in the channel made the changes

- Editing all levels where we displaying notes where we have jump link
- Separating chapter and section notes in partials

Use loose ruby version [TRELLO CARD](https://trello.com/c/YDtpanQs/477-tariff17-chapter-and-section-notes-need-to-be-more-clearly-highlighted)

### CODE REVIEW

**Comment by matthewford** [comment]
@jarl-borg think we can simplify this and just say there are important notes

**Comment by jarl-borg** [comment]
is that what you figured @matthewford  ?

**Comment by jarl-borg** [comment]
Don't have valid explanation why @matthewford , I was still learning the structure when I kinda started picking at this one. I hope this is what it needs to be now, had a lot of necessary code which I removed.

File: app/views/shared/_notes.html.erb [hunk 1]
```diff
@@ -3,13 +3,8 @@ chapter_note ||= false %>
...
 <% if section_note || chapter_note %>
   <article class="notes">
-
-    <% if section_note && chapter_note %>
-      <h2 class="heading-small">There are important section notes and chapter notes for this part of the tariff:</h2>
-    <% end %>
-
     <% if chapter_note %>
-      <div id="chapter-notes">
+      <div id="notes">
```
Comment (matthewford) [inline app/views/shared/_notes.html.erb]: think we can keep these and change the class above to an id and achieve the same thing, its best not to have two elements with the same id

File: app/views/shared/_notes.html.erb [hunk 2]
```diff
@@ -3,13 +3,8 @@ chapter_note ||= false %>
 
 <% if section_note || chapter_note %>
   <article class="notes">
```
Comment (matthewford) [inline app/views/shared/_notes.html.erb]: here add the id notes

**Comment by matthewford** [comment]
@jarl-borg no worries looking much cleaner, its good to go now

**Comment by jarl-borg** [comment]
@matthewford d'oh I didn't see that one. Thanks should be good now, pushing to dev to check it out there once more and have Craig bless it too

**Pull request merged**